packages/web-components/fast-foundation/src/tree-item/tree-item.template.ts

```diff
@@ -35,31 +39,34 @@ export function treeItemTemplate(
                 filter: elements(),
             })}
         >
-            <div class="positioning-region" part="positioning-region">
-                <div class="content-region" part="content-region">
-                    ${when(
-                        x => x.childItems && x.childItemLength(),
-                        html<FASTTreeItem>`
-                            <div
-                                aria-hidden="true"
-                                class="expand-collapse-button"
-                                part="expand-collapse-button"
-                                @click="${(x, c) =>
-                                    x.handleExpandCollapseButtonClick(
-                                        c.event as MouseEvent
-                                    )}"
-                                ${ref("expandCollapseButton")}
+            <div class="treeitem" part="treeitem">
+                ${when(
+                    x => x.childItems && x.childItemLength(),
+                    html<FASTTreeItem>`
+                        <div
+                            aria-hidden="true"
+                            class="expand-collapse-button"
+                            part="expand-collapse-button"
+                            @click="${(x, c) =>
+                                x.handleExpandCollapseButtonClick(c.event as MouseEvent)}"
+                            ${ref("expandCollapseButton")}
+                        >
+                            <span
+                                class="expand-collapse-icon"
+                                part="expand-collapse-icon"
                             >
```

Comment:
Why is this added span necessary? This feels like going backwards because we're now adding seemingly unnecessary elements to the template.

Reply:
Following the pattern where the slot is wrapped in a container element in order to target it for styling. In this case the `button` div is probably sufficient.

Following another set of discussions here, we can probably remove the slot for the icon and have the option only since this does not seem like content that is swapped at the instance level.